app/controllers/application_controller.rb

```diff
@@ -2,4 +2,22 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_action :configure_devise_permitted_parameters, if: :devise_controller?
+
+   protected
+
+  def configure_devise_permitted_parameters
+    registration_params = [:username, :email, :password, :password_confirmation]
+
+    if params[:action] == 'update'
+      devise_parameter_sanitizer.for(:account_update) {
+        |u| u.permit(registration_params << :current_password)
+        }
```

Comment:
`end` at 16, 8 is not aligned with `devise_parameter_sanitizer.for(:account_update) {` at 14, 6